readthedocs_ext/readthedocs.py

```diff
@@ -90,7 +91,10 @@ def update_body(app, pagename, templatename, context, doctree):
             pass
 
     if inject_css and theme_css not in app.builder.css_files:
-        app.builder.css_files.insert(0, theme_css)
+        if sphinx.version_info < (1, 8):
+            app.add_stylesheet(theme_css)
+        else:
+            app.add_css_file(theme_css)
```

Comment:
This may have a different effect since we were adding our .css file as the first element of the list.

So, all the css files added after our could override our own css classes. I'm not sure at what point `add_stylesheet` and/or `add_css_file` will add them.

In any case, we should check if inserting it at `0` position is something **we really need**.

Reply:
I saying this because I think the order of the list is the order in which the css files are loaded in the HTML, but I'm not sure either.

Reply:
We don't need that for recent projects, users can always pin the extension

https://github.com/rtfd/readthedocs-sphinx-ext/pull/57#issuecomment-451284885

Reply:
So, this is a breaking change, right?

If so, we won't be able to upgrade our default version installed by RTD.

Reply:
It's _breaking_ only for really old projects (sphinx_rtd_theme < 4.0, 2013), and supposing that they haven't updated the theme (they had the theme pinned to a version less than 4.0). And if they have a custom css rule that could interfere with the theme. I'm not really so worry about this change.

Reply:
> We don't need that for recent projects, users can always pin the extension

Big -1 here. Users should never need to know about our extension, asking them to pin it is definitely terrible UX :)